test/e2e_node/resource_metrics_test.go

```diff
@@ -165,3 +165,18 @@ func boundedSample(lower, upper interface{}) types.GomegaMatcher {
 		"Histogram": gstruct.Ignore(),
 	}))
 }
+
+func haveKeys(keys ...string) types.GomegaMatcher {
+	gomega.Expect(keys).ToNot(gomega.BeEmpty())
```

Comment:
I'd use `ExpectWithOffset` - not sure if there's a better method mark this function as helper

Reply:
Thanks, done :+1: